BUILDING.md

```diff
@@ -0,0 +1,99 @@
+## Building
+
+You need to set PICO_SDK_PATH in the environment, or pass it to cmake with `-DPICO_SDK_PATH=/path/to/pico-sdk`. To use features such as signing or hashing, you will need to make sure the mbedtls submodule in the SDK is checked out - this can be done by running this from your SDK directory.
+
+```console
+git submodule update --init lib/mbedtls
+```
+
+You also need to install `libusb-1.0` if you want to use the USB functionality.
+
+### Linux / macOS
+
+Use your favorite package tool to install dependencies. For example, on Ubuntu:
+
+```console
+sudo apt install build-essential pkg-config libusb-1.0-0-dev cmake
+```
+
+> If libusb-1.0-0-dev is not installed, picotool still builds, but it omits all options that deal with managing a pico via USB (load, save, erase, verify, reboot). Builds that do not include USB support can be recognized because these commands also do not appear in the help command. The build output message 'libUSB is not found - no USB support will be built' also appears in the build logs.
```

Comment:
Should this be moved out of the "Linux / macOS" section, and into the previous section where it  says "You also need to install `libusb-1.0` if you want to use the USB functionality." ?